src/style.css

```
@@ -0,0 +1,13 @@
+@import "./stars.css";
+@import "./card.css";
+
+.html {
+  font-family: "Roboto", sans-serif;
+}
```

I guess it's better to use tag selector here and you can also set  default font-size, color and   line-height and then change them where it's  needed
```suggestion    { html {
  font-family: Roboto, sans-serif;
  color: #060b35;
  font-size: 10px;
  line-height: 14px;
}
```